entries/w/w3schools.com.json

```
@@ -0,0 +1,12 @@
+{
+  "W3Schools": {
+    "domain": "w3schools.com",
+    "contact": {
+      "email": "support@w3schools.com",
+      "facebook": "w3schoolscom"
```

There's something strange with the w3schools Facebook page. I'll edit the Facebook script so that it can handle future exceptions like this. In the meantime, I suggest you change the handle to the following:
```suggestion
      "facebook": "374044253739"
```